manager/src/lib.rs

```diff
@@ -170,12 +171,34 @@ impl CronManager {
         self.slots.get(&current_slot).unwrap_or(default_vec)
     }
 
+    /// Most useful for debugging at this point.
+    pub fn get_all_tasks(&self, slot: Option<U128>) -> Vec<Base64VecU8> {
+        let mut ret: Vec<Base64VecU8> = Vec::new();
+        if let Some(slot_number) = slot {
+            // User specified a slot number, only return tasks in there.
+            let tasks_in_slot = self.slots.get(&slot_number.0).expect("Couldn't find tasks for given slot.");
+            for task in tasks_in_slot.iter() {
+                ret.push(Base64VecU8::from(task.to_vec()));
+            }
+        } else {
+            // Return all slots
+            for slot in self.slots.iter() {
+                let tasks_in_slot = slot.1;
+                for task in tasks_in_slot.iter() {
+                    ret.push(Base64VecU8::from(task.to_vec()));
+                }
+            }
+        }
+        ret
+    }
+
     /// Gets the data payload of a single task by hash
     ///
     /// ```bash
     /// near view cron.testnet get_task '{"task_hash": [0,102,143...]}' --accountId YOU.testnet
```

Comment:
@mikedotexe slight nit pick -- can you change this example to a base64 hash too?

Reply:
oh yeah, good catch